bor/beginblocker.go

```diff
@@ -22,7 +21,7 @@ func BeginBlocker(ctx sdk.Context, _ abci.RequestBeginBlock, k Keeper) {
 			return
 		}
 
-		var spans []*bor.ResponseWithHeight
+		var spans []*types.ResponseWithHeight
```

Comment:
why don't keep it in Bor package or, if needed, bor/api ?

Reply:
That creates dependency issue, I tried.
Then moved it to heimdall repo and it worked.

Reply:
Even a new package like bor/api?

Reply:
Okay, Let me try that

Reply:
While I was checking this I noticed that in latest bor code (0.3.x) we have removed `ResponseWithHeight` from the code, There is no struct like that anymore. I am looking more analyse the impact of that on heimdall code.

Reply:
So in latest version of bors we don't have these structs which I have replaced, So I think we will anyway have to update these after we release v0.3.0 on bor and update that in heimdall. I will create a task for that but for now I think the change I did should be fine.